examples/petsc/multigrid.c

```diff
@@ -173,6 +149,20 @@ int main(int argc, char **argv) {
     }
   }
 
+  VecType vectype;
+  switch (memtypebackend) {
+  case CEED_MEM_HOST: vectype = VECSTANDARD; break;
+  case CEED_MEM_DEVICE: {
+    const char *resolved;
+    CeedGetResource(ceed, &resolved);
+    if (strstr(resolved, "/gpu/cuda")) vectype = VECCUDA;
+    else if (strstr(resolved, "/gpu/hip")) vectype = VECHIP;
```

Comment:
Same

Reply:
Hmm, this is skipped in `junit.py`, but was removed from `tap.sh` in #305. I'll enable it in `junit.py` and mask it here.

Reply:
Done.